src/org/robockets/steamworks/drivetrain/ResetDriveEncoders.java

```diff
@@ -0,0 +1,35 @@
+package org.robockets.steamworks.drivetrain;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.steamworks.Robot;
+import org.robockets.steamworks.RobotMap;
+
+/**
+ * @author Jake Backer
+ */
+public class ResetDriveEncoders extends Command {
```

Comment:
It's used for a SmartDashboard button to fix things that occasionally break because of encoders